app/uploaders/validate_file_size.rb

```diff
@@ -8,6 +8,11 @@ module ValidateFileSize
   end
 
   def validate_file_size(file)
```

Comment:
this file doesn't have a spec.  I took a stab at making one in my carrierwave update PR https://github.com/department-of-veterans-affairs/vets-api/blob/2b9e53b0cc86d525fffa93c95decfe2ebf82c91d/spec/uploaders/validate_file_size_spec.rb